src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -222,26 +264,75 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                 Return valueText
             End Function
 
-            Private Shared Function CreateFloatingLiteralToken(token As SyntaxToken, newValueString As String, newValue As Double) As SyntaxToken
-                Debug.Assert(token.Kind = SyntaxKind.FloatingLiteralToken)
+            Private Function GetIntegerLiteralValueString(value As Object, base As LiteralBase) As String
+                Select Case base
+                    Case LiteralBase.Decimal
+                        Return CType(value, ULong).ToString(CultureInfo.InvariantCulture)
+                    Case LiteralBase.Hexadecimal
+                        Dim valueString = ""
+                        If TypeOf (value) Is Short Then
```

Comment:
Is there a reason this pattern is different than the pattern for hex? One you convert to ULong and then convert the string once. If we can unify, then it might be nice having a single helper for value -> ULong.

If there's a reason this has to be different, then a comment might be appreciated as to why the difference matters.